ModularTegustation/fishing/code/fishing_items/fishing_net.dm

```diff
@@ -144,7 +144,7 @@
 	icon_state_fished = "trawling_net_bait_full"
 	debris = list(/obj/item/fishing_net/baited = 1)
 	net_type = /obj/item/fishing_net/baited
-	fishin_cooldown = 15 SECONDS
+	fishin_cooldown = 15 SECONDS	//Slower.
```

Comment:
Please never use tabs for comments
```suggestion
	fishin_cooldown = 15 SECONDS //Slower.
```

Reply:
Wait, But all of the comments here are tabbed.